genon/preprocessor/facade/convert_processor.py

```
@@ -2941,118 +3128,174 @@ async def _process_request(self, request: Request, file_path: str, **kwargs: dic
-            # await assert_cancelled(request)
+    async def _process_ppt(self, request: Request, file_path: str, **kwargs: dict):
+        """PPT(.ppt)는 langchain 로더로 처리하고 페이지 이미지 메타를 부착한다."""
+        documents = self.load_documents_langchain(file_path, **kwargs)
+        chunks = self.split_documents_langchain(documents, **kwargs)
 
-            vectors = self.compose_vectors_langchain(chunks, file_path, **kwargs)
+        pdf_path = _get_pdf_path(file_path)
+        page_image_meta = {}
+        try:
+            page_image_meta = await self._extract_page_images(pdf_path, request)
+        except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)